docs/pages/reference/backends.mdx

```diff
@@ -620,6 +620,282 @@ Config option priority is applied in the following order:
+            ],
+            "Effect": "Allow",
+            "Resource": [
+                "arn:aws:s3:::audit-transient",
+                "arn:aws:s3:::audit-long-term"
+            ],
+            "Sid": "AllowListingMultipartUploads"
+        },
+        {
+            "Action": [
+                "s3:PutObject",
+                "s3:ListMultipartUploadParts",
+                "s3:GetObjectVersion",
+                "s3:GetObject",
+                "s3:DeleteObjectVersion",
+                "s3:DeleteObject",
+                "s3:AbortMultipartUpload"
+            ],
+            "Effect": "Allow",
+            "Resource": [
+                "arn:aws:s3:::audit-transient/results/*",
+                "arn:aws:s3:::audit-transient/large_payloads/*",
+                "arn:aws:s3:::audit-long-term/events/*"
+            ],
+            "Sid": "AllowMultipartAndObjectAccess"
+        },
+        {
+            "Action": "sns:Publish",
+            "Effect": "Allow",
+            "Resource": "arn:aws:sns:eu-central-1:1234567890:audit-sqs",
+            "Sid": "AllowPublishSNS"
+        },
+        {
+            "Action": [
+                "sqs:ReceiveMessage",
+                "sqs:DeleteMessage"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:sqs:eu-central-1:1234567890:audit-sns",
+            "Sid": "AllowReceiveSQS"
+        },
+        {
+            "Action": [
+                "glue:GetTable",
+                "athena:StartQueryExecution",
+                "athena:GetQueryResults",
+                "athena:GetQueryExecution"
+            ],
+            "Effect": "Allow",
+            "Resource": [
+                "arn:aws:glue:eu-central-1:1234567890:table/audit_db/audit_table",
+                "arn:aws:glue:eu-central-1:1234567890:database/audit_db",
+                "arn:aws:glue:eu-central-1:1234567890:catalog",
+                "arn:aws:athena:eu-central-1:1234567890:workgroup/audit_workgroup"
+            ],
+            "Sid": "AllowAthenaQuery"
+        },
+        {
+            "Action": [
+                "kms:GenerateDataKey",
+                "kms:Decrypt"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:kms:eu-central-1:1234567890:key/kms_id",
+            "Sid": "AllowAthenaKMSUsage"
+        }
+    ]
+}
+```
+
+### Migration from Dynamo to Athena audit logs backend
+
+<Admonition
+  type="tip"
+  title="Tip"
+>
+  Migration is only needed if Dynamo audit logs was used and you want to keep
+  old data.
+</Admonition>
+
+Migration consist of following steps:
+
+1. Set up Athena infrastructure
+1. Dual write to both Dynamo and Athena, query from Dynamo
+1. Migrate old data from Dynamo to Athena
+1. Dual write to both Dynamo and Athena, query from Athena
+1. Disable writing to Dynamo
+
+In teleport storage config, `audit_events_uri` accepts multiple URLs used as
+audit loggers. If more than 1 is used, then events are written to each audit
```

Comment:
We should rewrite "URLs used as audit loggers", since the URLs themselves are not being used as audit loggers. Instead, Teleport is using these URLs to configure its connections to the different audit loggers, right?

Reply:
In teleport storage config, `audit_events_uri` accepts multiple URLs. Those URLs are used to configure connections to the different audit loggers. 

Is it better?

Reply:
Yes, thanks!